src/main/java/no/uib/inf112/core/player/Player.java

```diff
@@ -16,15 +16,18 @@
     public static final int MAX_LIVES = 3;
     public static final int MAX_HEALTH = 10;
     public static final int MAX_PLAYER_CARDS = 5;
+    public static final int MAX_DRAW_CARDS = 9;
 
     private Robot robot;
-    private Deck deck; //TODO Issue #33 move to PlayerHandler
 
     private Vector2Int backup;
 
+    private int dock;
+
     private int lives;
     private boolean poweredDown;
     private int health;
+    private int damageTokens;
```

Comment:
Damage tokens seem to be the same as `MAX_HEALTH - health - 1` (please comment if there is a different)

Reply:
Why -1? That would result in -1 damage tokens at full health.

Reply:
You are right it should just be `MAX_HEALTH - health`